dialogflow/stopwatch/README.md

```diff
@@ -1,42 +1,72 @@
+- Via Text
+- Via Streaming Audio
+- With Sentiment Analysis
+- With Text-to-Speech
+- With Knowledge Connectors
+
+To call the Dialogflow API from iOS, you need to provide authorization tokens with each request for them to be accepted by the Dialogflow API. To get this token, this sample uses a Firebase Function (in Node.js) to generate these tokens on the behalf of a service account to be used by the app when making a request to the Dialogflow API.
 
 ## Prerequisites
-- Credentials for calling the Dialogflow API
-- An OSX machine or emulator
+- An OSX machine or simulator
 - [Xcode 9.1][xcode] or later
 - [Cocoapods][cocoapods] version 1.0 or later
 
-## Prepare your Google Cloud account
-As with all Google Cloud APIs, every call to the Dialogflow API must be associated
-  with a project within the [Google Cloud Console][cloud-console] that has the
-  API enabled. In brief:
-  - Create a project (or use an existing one) in the [Cloud Console][cloud-console]
-  - [Enable billing][billing].
-
-## Enable Dialogflow
-If you have not already done so, [enable Dialogflow for your project](https://cloud.google.com/dialogflow-enterprise/docs/quickstart). Scripts related to the quickstart are in the SETUP-SERVICE directory.
-
-## Running the app
-- In the [Google Cloud Console](https://console.cloud.google.com), use the APIs&Services->Library menu item to find and enable the Dialogflow API.
-- Also using the Credentials item in the Google Cloud Console, create a Service Account key. Use the popup to create a key for a "New service account", name it "stopwatch" (this is arbitrary), and give it the role of `Dialogflow API Admin`. Choose a key type of "JSON" and press the "Create" button to download a key file. As a convenience, rename this file to `credentials.json`.
-- Clone this repository and `cd` into this directory.
-- Copy your `credentials.json` over the skeletal version in this directory. It is used by the scripts in `SETUP_SERVICE` and by the sample application. 
-- Be sure that you have gone through the steps in the [Dialogflow quickstart](https://cloud.google.com/dialogflow-enterprise/docs/quickstart) to create and configure your stopwatch agent. Helper scripts for this are in the `SETUP_SERVICE` directory.
-- Run ./INSTALL-COCOAPODS to install app dependencies. When it finishes, it will open the Stopwatch workspace in Xcode. Since we are using Cocoapods, be sure to open the workspace and not Stopwatch.xcodeproj.
-- Replace `your-project-identifier` in `StopwatchService.swift` with the identifier of your Google Cloud project.
+## Setup
+- Create a project with the [Google Cloud Console][cloud-console]
+- Enable the [Dialogflow API](https://console.cloud.google.com/apis/librarydialogflow.googleapis.com).
+- [Enable billing][billing].
+- [Import the Dialogflow Agent](https://dialogflow.com/docs/agents/export-import-restore#import) using the `StopwatchAgent.zip` which is located in the `stopwatch` directory. 
```

Comment:
I don't follow this step. The linked instructions seem to assume that I'm at a particular place in the Dialogflow console, but I don't see how to get there. I think we're likely to lose a lot of users if we can't make this more clear.